uxarray/cross_sections/dataarray_accessor.py

```diff
@@ -17,60 +17,84 @@ def __repr__(self):
         prefix = "<uxarray.UxDataArray.cross_section>\n"
         methods_heading = "Supported Methods:\n"
 
-        methods_heading += "  * constant_latitude(center_coord, k, element, **kwargs)\n"
+        methods_heading += "  * constant_latitude(lat, use_spherical_bounding_box)\n"
+        methods_heading += "  * constant_longitude(lon, use_spherical_bounding_box)\n"
 
         return prefix + methods_heading
 
-    def constant_latitude(self, lat: float, use_spherical_bounding_box=False):
-        """Extracts a cross-section of the data array at a specified constant
-        latitude.
+    def constant_latitude(self, lat: float, use_spherical_bounding_box: bool = True):
+        """Extracts a cross-section of the data array by selecting all faces that
+        intersect with a specified line of constant latitude.
 
         Parameters
         ----------
         lat : float
             The latitude at which to extract the cross-section, in degrees.
+            Must be between -90.0 and 90.0
         use_spherical_bounding_box : bool, optional
-            If True, uses a spherical bounding box for intersection calculations.
+            If True, uses a spherical bounding box for obtaining candidate faces, which considers the extreme cases
```

Comment:
what does the spherical bounding box mean here?

Reply:
`use_spherical_box` is used to select whether `Grid.bounds` should be used for performing the cross sections instead of checking every edge of every face using a naive approach.